test/Google.Api.Gax.Tests/PageStreamingRequestPartial.cs

```diff
@@ -0,0 +1,15 @@
+﻿/*
+ * Copyright 2016 Google Inc. All Rights Reserved.
+ * Use of this source code is governed by a BSD-style
+ * license that can be found in the LICENSE file or at
+ * https://developers.google.com/open-source/licenses/bsd
+ */
+
+// Namespace of gRPC codegen
+namespace Google.Api.Gax
+{
+    public partial class PageStreamingRequest : IPageRequest
+    {
+        // No code required, the proto code already implements IPageRequest
```

Comment:
I mean that PageStreamingRequest already defines the properties PageToken and PageSize as required by IPageRequest, so it's not possible to define them here.